client/src/style/CourseSelection.css

```diff
@@ -11,6 +11,17 @@
 .courses h2{
   text-align:center;
 }
+
+.center {
+  position: absolute;
+  margin-top: 22%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  width: 45vw;
+  color: #ffffff;
+}
+
```

Comment:
What impact does this have on the mapped cards from:

https://github.com/oscarso2000/MyCourseIndex/blob/76689cab401bb649d37a69099a6921ae315683c2/client/src/components/CourseSelection.tsx#L31-L48

Reply:
the margin-top is the most important cuz there was a really weird bug that was shifting the card modals around so i had to find the exact location given screen-resizing to put the course cards.

Reply:
Screenshot to show the impact and what the original thing was?

Reply:
<img width="1440" alt="Screen Shot 2020-09-18 at 3 29 00 PM" src="https://user-images.githubusercontent.com/20172573/93637479-af1be000-f9c3-11ea-9a0d-12b23498b14f.png">

Reply:
<img width="1440" alt="Screen Shot 2020-09-18 at 3 30 47 PM" src="https://user-images.githubusercontent.com/20172573/93637662-edb19a80-f9c3-11ea-97c6-1c509146e99d.png">
fixed version w/ css